tests/PHPStan/Analyser/data/bug-2600.php

```diff
@@ -65,21 +65,21 @@ public function doBar($x = null) {
 	 * @param string $x
 	 */
 	public function doBaz(...$x) {
-		assertType('array<int, string>', $x);
+		assertType('list<string>', $x);
 	}
 
 	/**
 	 * @param string ...$x
 	 */
 	public function doLorem(...$x) {
-		assertType('array<int, string>', $x);
+		assertType('list<string>', $x);
 	}
 }
 
 function foo($x, string ...$y): void
 {
 	assertType('mixed', $x);
-	assertType('array<int, string>', $y);
+	assertType('list<string>', $y);
 }
 
 function ($x, string ...$y): void {
```

Comment:
why is L87 not a list, is that correct?

Reply:
ah I guess the same thing needs to be adapted for closures for that or something like that.